### PR TITLE
feat(documentation): parallel gateway

### LINKED ISSUE #1444: I can read documentation on parallel gateway
* including sequence flow mappings

### PR DESCRIPTION
**Pull request by ThorbenLindhauer**
closes #1444

### CODE REVIEW

File: docs/src/bpmn-workflows/gateways.md [hunk 1]
```diff
@@ -32,3 +33,26 @@ XML representation:
 ```
 
 BPMN Modeler: [Click Here](/bpmn-modeler/gateways.html#create-an-exclusive-gateway)
+
+## Parallel Gateway (AND)
+
+![workflow](/bpmn-workflows/parallel-gateway-example.png)
+
+A parallel gateway is only activated, when each of its incoming sequence flows has been signalled. Once activated, it signals all of its outgoing sequence flows.
```
Comment (saig0) [inline docs/src/bpmn-workflows/gateways.md]: Can you describe more in detail how a parallel gateways works? How is the fork working? What happens to the payload?

Also, I think it's a bit confusing when you talk about that a sequence flow is signaled.

**Review by saig0 — CHANGES_REQUESTED** [review]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 02043168d81ae7743795d2464d898d75780b4424
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, ci, build**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Comment by ThorbenLindhauer** [comment]
@saig0 I expanded the explanation.

**Review by saig0 — APPROVED** [review]

**Pull request merged**